Cargo.toml

```
@@ -11,6 +11,7 @@ ip_who_provider=[]
 usa_local_test=["ip_info_provider", "ip_who_provider"]
 
 [dependencies]
+clap = { version = "3.1.18", features = ["derive"] }
```

Clap is now at `3.2.6`
```suggestion
clap = { version = "3.2.6", features = ["derive"] }
```